scripts/doxygen.sh

```diff
@@ -21,6 +21,9 @@ if [ ! $(version "$DOXYGEN_VERSION") -eq $(version "1.9.1") ] ; then
   exit 0
 fi
 
+export RAPIDS_VERSION="$(sed -E -e 's/^([0-9]{2})\.([0-9]{2})\.([0-9]{2}).*$/\1.\2.\3/')"
+export RAPIDS_VERSION_MAJOR_MINOR="$(sed -E -e 's/^([0-9]{2})\.([0-9]{2})\.([0-9]{2}).*$/\1.\2/')"
```

Comment:
Were these sed commands supposed to be replaced by `rapids-version` and `rapids-version-major-minor`?

Reply:
No. Because this file can be run outside of CI, it can't rely on `gha-tools`.

Reply:
Though I just noticed the `sed` commands are not actually reading from `VERSION`. Fixed.

Reply:
Great! Okay, that part about `VERSION` was my second question :)